Nos/Views/ActionButton.swift

```diff
@@ -128,7 +128,7 @@ struct ActionButtonStyle: ButtonStyle {
                 .padding(.vertical, 8)
                 .padding(.horizontal, 13)
                 .shadow(
-                    color: textShadow ? Color(white: 0, opacity: 0.15) : .clear,
+                    color: textShadow ? Color.actionButtonTextShadow : .clear,
```

Comment:
If I'm reading this right, white should be `0` so I'd expect the hex to be `#000000`. But in the asset, the hex is `#FFFFFF`.

Reply:
When I changed these types of colors, I used Xcode to change the input method in the properties pane from RGB or white, opacity to hex. I will take another look at this and see if I might have changed something unintentionally.